src/components/browse/dataset/datasetAccordion/datasetSummary/datasetStudies.tsx

```diff
@@ -16,17 +16,18 @@ const DatasetStudies = (props: dataSetStudiesProps) => {
         props.study !== null ? (
           <div>
             <p className="mb-0">
-              <strong>
-                Part of study:&nbsp;
-              </strong>
-              {props.study.stats.title !== null ? props.study.stats.title : props.study.stats?.accession}
-
-              <br />
+              <strong>Part of study:&nbsp;</strong>
+              {props.study.stats.title !== null
```

Comment:
Just a side node, this is is now easier with
```ts
props.study.stats?.title ?? props.study.stats?.accession
```

Reply:
This would require a lot of refactoring for a lot of files, since it's present in a lot of places, even within this file, which I'd rather do separately.

Reply:
Sure, I did not mean that needs to be changed everywhere now.